Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -2146,3 +2114,18 @@ def uninstallPipeline2(self):
         elif self.vis_mode == CILViewer2D.RECTILINEAR_WIPE:
             # rectilinear wipe visualises 2 images in the same pipeline
             pass
+
+    def getSliceMapWindow(self, percentiles):
+        ia = vtk.vtkImageHistogramStatistics()
```

Comment:
This is not giving the map window for the slice, it is giving it for the whole 3D dataset.

Reply:
So this is just a bad name for the function, effectively it's doing the same thing as the Volume namesake in the none-2D viewer, it's been renamed to slice here because it's 2D alone. Perhaps I should rename it to something like `getAllSliceMapWindow`?

Reply:
Currently in the 2D viewer we adjust the window/level based on the values in the current slice, not in the whole image. So auto window-level just takes into account the values on that slice, rather than in the whole volume. In the 3D viewer it's based on the whole volume.

We want the 2D and 3D viewer to perform in the same way for the slice, so the question is which do we want them both to do?
@paskino what do you think?
This can be resolved in a separate issue/PR though

Reply:
It gives the raw value (867, 867)